### PR TITLE
Changelog for Code eval packages, structured output and changes to Stdout

### PR DESCRIPTION
**Pull request by gavinfd-hl**
ref ENG-982

### CODE REVIEW

File: fern/changelog/2024-08-07.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Additional Python Packages for Code Evaluators
+
+We're excited to annouce have expanded the default python packages available in the environment for Code Evaluator. The full list of packages now supported are: `anthropic`, `continuous-eval`, `jellyfish`, `nltk`, `numpy`, `openai`, `pandas`, `requests`, `scikit-learn`, `spacy`, `syllapy`, `textstat`, `transformers`, `jsonschema`, `sqlglot`.
```
Comment (PeadarOhAodha) [inline fern/changelog/2024-08-07.md:3]: Generally dislike 'we're excited'

Comment (PeadarOhAodha) [inline fern/changelog/2024-08-07.md:3]: 'in the environment for Code Evaluator' reads awkwardly

File: fern/changelog/2024-08-07.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+We are actively improving our execution environment so if you have additional packages you'd like us to support, please do not hesitate to get in touch.
```
[the remark this answers is not in the document]
Comment (gavinfd-hl) [reply fern/changelog/2024-08-07.md:5]: I added the full list so your comment to Vanta doesnt seem strange. If we only call out the ones we've added it doesnt align with what you said, as 3 or 4 of those were already there.

> the only one we are not adding from this list is torch

[the remark this answers is not in the document]
Comment (gavinfd-hl) [reply fern/changelog/2024-08-07.md:5]: But sure.

[the remark this answers is not in the document]
Comment (PeadarOhAodha) [reply fern/changelog/2024-08-07.md:5]: Happy to tell them some of them already exist

File: fern/changelog/2024-08-07.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+## Additional Supported Python Packages
+
+We have expanded the packages available in the Evaluator Python environment. The new packages we've added are: `continuous-eval`, `jellyfish`, `langdetect`, `nltk`, `scikit-learn`, `spacy`, `transformers`. The full list of packages can been seen in our [Python environment reference](https://humanloop.com/docs/v5/reference/python-environment).
```
Comment (harry-humanloop) [inline fern/changelog/2024-08-07.md:3]: can/should the link be a relative one?

Comment (gavinfd) [reply fern/changelog/2024-08-07.md:3]: I dont know, do you know how that's been done on previous changelogs?

Comment (gavinfd-hl) [reply fern/changelog/2024-08-07.md:3]: The most recent place I can find this being done is in [here](https://github.com/humanloop/humanloop-docs/commit/7007be27caee33ddbd43c6093962f16acb53616a#diff-72550c2b68e159326b6e14db9f707d5b74c4af59e82d09c87de9ebc847fadb11R3)

Comment (gavinfd-hl) [reply fern/changelog/2024-08-07.md]: Seems to work with the relative link, will leave it as that

**Comment by gavinfd-hl** [comment]
✅ LGTM

**Pull request merged**